doc/release/1.13.0-notes.rst

```diff
@@ -4,20 +4,31 @@ NumPy 1.13.0 Release Notes
 
 This release supports Python 2.7 and 3.4 - 3.6.
 
-  
+
 Highlights
 ==========
 
  * Operations like ``a + b + c`` will reuse temporaries on some platforms,
    resulting in less memory use and faster execution.
  * Inplace operations check if inputs overlap outputs and create temporaries
    to avoid problems.
- * Improved ability for classes to override default ufunc behavior. See
-   ``__array_ufunc__`` below.
+ * New ``__array_ufunc__`` attribute provides improved ability for classes to
+   override default ufunc behavior.
+ * New ``np.block`` function for creating blocked arrays.
+
 
+New functions
+=============
 
-Dropped Support
-===============
+* New ``np.positive`` ufunc provides override for ``__pos__``.
```

Comment:
This isn't true yet, unfortunately -- see https://github.com/numpy/numpy/pull/8967#issuecomment-299955657

So either we'll want to get that deprecation warning/override in, or change this note